cmake/Modules/ObsHelpers.cmake

```diff
@@ -337,38 +337,96 @@ function(define_graphic_modules target)
   endforeach()
 endfunction()
 
+if(NOT QT_VERSION)
```

Comment:
I'm not sure if this check is correct (given CMake's stringly typed nature).

In _theory_ `if(NOT ..)` checks for the boolean value of a variable. _If_ Qt considers a non-defined variable `false`, this would work. Otherwise (and to be specific) this would need to say:

`if(NOT DEFINED QT_VERSION OR QT_VERSION STREQUAL "")` which ensures that you only `FORCE` the cache variable if it is not defined at all or contains an empty string.

Reply:
> `if(<constant>)`
> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number (including floating point numbers). False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. 

Script languages, you gotta love 'em. So I guess the code _works_ fine.

Reply:
The next push will have `NOT DEFINED` for each `NOT QT_VERSION` or `NOT _QT_VERSION`. And also `QT_VERSION STREQUAL ""` will  be added.

Reply:
So I will keep it like that.